Containerfile.marin3r-operator

```diff
@@ -1,6 +1,19 @@
 # Based on ./gatekeeper-operator/Dockerfile
 # Build the manager binary
-FROM mirror.gcr.io/library/golang:1.24 as builder
+#FROM mirror.gcr.io/library/golang:1.24 as builder
+
+# ----- replace mirror.gcr.io/library/golang:1.24
+FROM debian:bullseye-slim as builder
```

Comment:
what's the reason for these changes?

Reply:
the reason of these changes - go1.24 toolchain not available. Error:
2025-05-06 11:26:07,354 INFO Fetching the gomod dependencies at subpath marin3r/api
2025-05-06 11:26:07,699 ERROR The command "go env GOWORK" failed
2025-05-06 11:26:07,699 ERROR STDERR:
go: downloading go1.24 (linux/amd64)
go: download go1.24 for linux/amd64: toolchain not available
2025-05-06 11:26:08,096 ERROR PackageManagerError: Go execution failed: `go env GOWORK` failed with rc=1
Error: PackageManagerError: Go execution failed: `go env GOWORK` failed with rc=1